relay/workflow.py

```diff
@@ -21,7 +21,7 @@
 # without a specific handler, it will log to the console. Uncomment below to not send to console.
 # logger.addHandler(logging.NullHandler())
 
-version = "relay-sdk-python/2.0.1"
+version = "relay-sdk-python/2.0.0"
```

Comment:
should version be incremented?

Reply:
Thought is to bump the version at the time of release. At that point we'll see what kind of content we have (hotfix vs addition vs breaking change), so we can figure out what the new version number should be.

Reply:
ah, so this is just reverting a previous 2.0.1?  got it.